burn-train/src/metric/acc.rs

```diff
@@ -85,6 +85,69 @@ impl<B: Backend> Numeric for AccuracyMetric<B> {
     }
 }
 
+/// The Image accuracy metric.
+#[derive(Default)]
+pub struct ImageAccuracyMetric<B: Backend> {
+    state: NumericMetricState,
+    threshold: f64,
+    _b: B,
```

Comment:
I recommend you use [`PhantomData`](https://doc.rust-lang.org/std/marker/struct.PhantomData.html#unused-type-parameters) to wrap `B`.